src/services/sessionInvite.js

```diff
@@ -326,6 +328,13 @@ module.exports = class UserInviteHelper {
 				} else {
 					session.mentor_id = mentorId.result.id
 				}
+				if (
+					session.type == common.SESSION_TYPE.PRIVATE &&
+					!session.mentees.some((item) => typeof item === 'number')
```

Comment:
Will this work if the mentee UID is email ?

Reply:
i am checking it after fetching the mentee id , so i have also set that if mentee exists replace with mentee id and if the mentee is invalid send the same string .